usda_nutrition/management/commands/import_usda.py

```diff
@@ -1,6 +1,11 @@
 import csv
 import os
 import sys
+if (sys.version_info < (3, 0)):
+    reload(sys)
+    sys.setdefaultencoding('utf-8')
```

Comment:
Hi, thanks for the pull request! I'm good with the other changes, but would it work for you if you added the encoding to the modules that are causing issues rather than globally set the encoding? `models.py` I presume? e.g., like this: https://docs.python.org/2.3/whatsnew/section-encodings.html